index.html

```diff
@@ -1,15 +1,36 @@
-<html>
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width , initial-scale=1.0">
+    <title>Uma Venki</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Dosis:wght@200..800&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="css/index.css"/>
+</head>
+<body>
+    <section id="name">
     <h1>Uma Venki</h1>
-    <h2>About</h2>
-    <p>I'm a student of Code the dream.I'm learning Intro to programing.</p>
-    <h2>Qualifications</h2>
-    <ul>
-        <li>Bachelor Of Engineering</li>
-    </ul>
-    <h2>Connect</h2>
-    <ul>
-        <li>Linkedln</li>
-        <li><a href="https://github.com/umavenki/uma-venki-haumea">Github</a></li>
+    </section>
+    <section id="About">
+        <h2>About</h2>
+        <p>I'm a student of Code the dream.I'm learning Intro to programing</p>
+    </section>
+    <section id="Experience">
 
-    </ul>
+        <h2>Experience</h2>
+        <ul>
+            <li>Four years of learning Bachelor Of Engineering in Anna Unversity</li>
+        </ul>
+    </section>
+    <section id="Connect">
+        <h2>Connect</h2>
+        <ul>
+            <li><img src="/Users/umasekar/Documents/Uma/CodeTheDream/haumea-repos/uma-venki-haumea/Images/5296501_linkedin_network_linkedin logo_icon.svg"><a href="https://www.linkedin.com/in/uma-sekar-4073982a4">Linkedln</a></li>
+            <li><img src="/Users/umasekar/Documents/Uma/CodeTheDream/haumea-repos/uma-venki-haumea/Images/211904_social_github_icon.svg"><a href="https://github.com/umavenki/uma-venki-haumea">Github</a></li>
```

Comment:
These src paths will only work on your machine.  You should use a relative path './Images/...'.